development/docker-compose.yml

```diff
@@ -1,24 +1,36 @@
-version: "3"
+---
+x-nautobot-build: &nautobot-build
+  build:
+    args:
+      NAUTOBOT_VER: "${NAUTOBOT_VER}"
+      PYTHON_VER: "${PYTHON_VER}"
+    context: "../"
+    target: "nautobot"
+    dockerfile: "development/Dockerfile"
+x-nautobot-base: &nautobot-base
+  image: "nautobot/nautobot:local"
+  env_file:
+    - "dev.env"
+  tty: true
+
+version: "3.8"
 services:
   nautobot:
-    image: "ghcr.io/nautobot/nautobot:${NAUTOBOT_VER}-py${PYTHON_VER}"
-    command: "nautobot-server runserver 0.0.0.0:8000"
+    command: "nautobot-server runserver 0.0.0.0:8000 --insecure"
     ports:
       - "8000:8000"
     depends_on:
       - "postgres"
       - "redis"
-    env_file:
-      - "./dev.env"
-    tty: true
+    <<: *nautobot-build
```

Comment:
I don't fully understand the need to create an anchor for lines that isn't used in more than one place....

Reply:
Sorry about that, quick and dirty copy-paste ;-). Updated accordingly.